api/app/app.py

```diff
@@ -0,0 +1,65 @@
+#app.py
+from flask import Flask, request, jsonify, abort
+from models import Books, Users
+import json
+
+def create_app():   
+    app = Flask(__name__, instance_relative_config=True)
+    app.config["TESTING"] = True
+
+    book = Books()
+
+    @app.route('/books', methods=['POST', 'GET'])
```

Comment:
This is a function inside a function, I dont think that is what you want here.

Reply:
This was the way I got rid of the test_client error in test_api.py